docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+
+The question of how many Data Landing Zones an organization requires should be discussed and defined upfront before adopting the ESA prescriptive architectural pattern. This is considered to be one of the most important design decisions, because it lays the foundation for an effective and efficient data platform. When done right, it will prevent enterprises from ending up in a migration project of data products and data assets from one Data Landing Zone to another and will allow an effective and consistent scaling of any big data and analytics efforts for the upcoming years.
+
+The following factors should be considered when deciding about how many Data Landing Zones should be deployed:
+
+- *Data Domains*: What data domains does the organization encompass and which data domains will land on the data platform?
+- *Cost allocation*: Are shared services like storage accounts paid centrally or do these need to be split by business unit or domain?
+- *Location*: In which Azure Regions will the organization deploy their data platform? Will the platform be hosted in a single region or will it span across multiple regions? Are there data residency requirements that need to be taken into account?
+- *Data classifications and highly confidential data*: What data classifications exist within the organization? Does the organization have datasets that are classified as highly confidential and do these datasets require special treatment in form of just in time access, customer managed keys (CMK), fine grained network controls or additional encryption being enforced? May these additional security mechanisms even impact usability of the data platform and data product development?
+
+Considering all these factors, an enterprise should target no more than 15 Data Landing Zones as there is also some management overhead attached to each Data Landing Zone.
+
+It is important to emphasize that Data Landing Zones are not creating Data Silos within an organization, as the recommended network setup in ESA enables secure and in-place data sharing across Landing Zones and therefore enables innovation across data domains and business units. Please read the [network design guidance](/docs/guidance/EnterpriseScaleAnalytics-NetworkArchitecture.md) to find out more about how this is achieved.
```

Comment:
I think we need to discuss this a bit more. When it comes to sharing data between domains, customers usually pivot the conversation to data virtualisation (in-place sharing of data) versus copying data across domains data landing zones (or in other words, duplication of data). I think it will add more value if we expand on this point. Whilst networking layout definitely enables exchange of data between domains, addressing virtualization v/s physically moving data between domains would help. 

Perhaps, we need to call out that with mesh architecture both virtualization and data copy between domains are a valid option?

Reply:
Maybe we should add a link to a doc about how data is shared between domains. You are right. It is not all about network. Network is a part of the setup.